src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1032,10 +1032,7 @@ private void ProcessOutputRecord(ProcessOutputObject outputValue)
 
             if (outputValue.Stream == MinishellStream.Error)
             {
-                ErrorRecord record = outputValue.Data as ErrorRecord;
-                Dbg.Assert(record != null, "ProcessReader should ensure that data is ErrorRecord");
-                record.SetInvocationInfo(this.Command.MyInvocation);
-                this.commandRuntime._WriteErrorSkipAllowCheck(record, isNativeError: true);
+                this.commandRuntime._WriteErrorSkipAllowCheck(outputValue.Data);
```

Comment:
Please add a comment stating that it is the intent to not wrap native stderror in an ErrorRecord.

Reply:
will add